libsolidity/ast/AST.h

```diff
@@ -753,6 +760,7 @@ class VariableDeclaration: public Declaration
 	bool m_isStateVariable; ///< Whether or not this is a contract state variable
 	bool m_isIndexed; ///< Whether this is an indexed variable (used by events).
 	bool m_isConstant; ///< Whether the variable is a compile-time constant.
+	bool m_isOverriding; ///< whether the variable is overriding a public base class variable
```

Comment:
```suggestion
	bool m_isOverriding; ///< whether the variable is overriding an external base contract function
```

Reply:
A state variable can override a function fro the base contract?

Reply:
The only situation where `override` is meaningful for state variables is if the getter of the state variable overrides a function, and for that, the function in the base contract has to be external.